src/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
+import { Provider, compose } from 'react-redux';
```

Comment:
@Spetastian This fixed part of it, but now I get:

```
Uncaught TypeError: h is not a function
    at Object.r.529../components/App (build-7ee3a451c8.min.js:40)
    at o (build-7ee3a451c8.min.js:16)
    at e (build-7ee3a451c8.min.js:16)
    at build-7ee3a451c8.min.js:16
r.529../components/App @ build-7ee3a451c8.min.js:40
o @ build-7ee3a451c8.min.js:16
e @ build-7ee3a451c8.min.js:16
(anonymous) @ build-7ee3a451c8.min.js:16
```

The code that `compose` transpiles to calls `h()` but it's not defined for some reason? Hmm...